evalml/tests/component_tests/test_lgbm_classifier.py

```diff
@@ -168,8 +162,7 @@ def test_correct_args(mock_fit, mock_predict, mock_predict_proba, X_y_binary):
 
 @patch('evalml.pipelines.components.estimators.estimator.Estimator.predict_proba')
 @patch('evalml.pipelines.components.estimators.estimator.Estimator.predict')
-@patch('evalml.pipelines.components.component_base.ComponentBase.fit')
-def test_categorical_data_subset(mock_fit, mock_predict, mock_predict_proba, X_y_binary):
+def test_categorical_data_subset(mock_predict, mock_predict_proba, X_y_binary):
```

Comment:
I had a hard time with these tests; I updated to remove the ComponentBase fit mock since it no longer calls that but wasn't sure if I was ripping out what the test was trying to test for in the first place.

My understanding is that it was trying to test if our encoding was passed to the super class' fit/predict/predict_proba implementations correctly. Since LGBM no longer calls the super class' impl, this isn't necessary?

Reply:
Yep, sounds good to me!